easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.17-GCC-11.2.0.eb

```diff
@@ -30,6 +33,14 @@ checksums = [
     'b1f89d90cc46b486534069135854e6081994b5a1a88370d32dae982e0f040111',  # OpenBLAS-0.3.17_fix-segfault-ztrsv.patch
 ]
 
+builddependencies = [
+    # required by LAPACK test suite
+    ('Python', '3.9.6', '-bare'),
+]
+
+run_lapack_tests = True
+max_failing_lapack_tests_num_errors = 150
```

Comment:
I'd set it to 1000 instead of 150.
There are some fairly benign tests still failing with -ftree-vectorize after GCC is fixed, but this way it still fails if things are really bad.

Basically the benign tests use CGEEV (etc.) with and without computation of eigenvectors (in both cases eigenvalues are computed), and compare the eigenvalues. If they're not numerically exactly the same, the tests fails, even if those eigenvalues are superclose. It'll take some time to sort those out but this shouldn't have real-world significance.